components/portal/widgets/directives.js

```diff
@@ -33,13 +33,26 @@ define(['angular', 'require'], function(angular, require) {
       },
       templateUrl: require.toUrl('./partials/widget-card.html'),
       controller: 'WidgetCardController',
+      link: {
+        post: function(scope, element, attrs) {
+          element.on('click', function(event) {
+            var el = event.target;
+            while (el && (!el.tagName || el.tagName.toLowerCase() !== 'a')) {
```

Comment:
:question: Will there always be a parent link tag?
I'd be surprised if there aren't some areas that are using a button.

Reply:
good catch. I just checked in widget-creator, almost everything is using an anchor, except for action items.

... that being said, I wonder if action items should actually have anchors instead.